### PR TITLE
homepage html structure

### LINKED ISSUE #2: Create HTML for homepage

### PR DESCRIPTION
**Pull request by JoannaTabbi**
Close #2

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+    <script
+      src="https://kit.fontawesome.com/b2f281879d.js"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+  <body>
+    <!--      HEADER     -->
+
+    <header>
+      <img src="images/logo/ohmyfood.png" alt="logo du site" />
```
Comment (Tutt-J) [inline index.html]: Don't fortget ./ before" image/..."

Alt must be "Logo de OhMyFood"

File: index.html [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+                src="images/restaurants/jay-wennington-N_Y88TWmGwA-unsplash.jpg"
+                alt="plat fin gastronomique et deux verres de vin"
+              />
+            </a>
+            <figcaption>
+              <a href="./pages/la-palette-du-gout.html">
+                <h3>La palette du goût</h3>
+                <p>Ménilmontant</p>
+              </a>
+              <i class="far fa-heart"></i>
```
Comment (Tutt-J) [inline index.html]: You have to add also heart plain and put the two i inside a span

File: index.html [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+            </figcaption>
+          </figure>
+        </article>
+      </section>
+    </main>
+
+    <!--     FOOTER     -->
+
+    <footer>
+      <div>
```
Comment (Tutt-J) [inline index.html]: Maybe better a nav

File: index.html [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+        <h4>ohmyfood</h4>
```
Comment (Tutt-J) [inline index.html]: Don't sure of h4

File: index.html [hunk 5]
```diff
@@ -0,0 +1,162 @@
...
+        <ul>
+          <li>
+            <a href=""><i class="fas fa-utensils"></i>Proposer un restaurant</a>
```
Comment (Tutt-J) [inline index.html]: Always put a # on href when there is nothing

**Review by Tutt-J — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (JoannaTabbi) [reply index.html]: ok done

File: index.html (hunk 2 above)
Comment (JoannaTabbi) [reply index.html]: ok got it

File: index.html (hunk 3 above)
Comment (JoannaTabbi) [reply index.html]: ok

File: index.html (hunk 4 above)
Comment (JoannaTabbi) [reply index.html]: h3 then ? as I'm not sure that this title is as important as others sections titles...

File: index.html (hunk 5 above)
Comment (JoannaTabbi) [reply index.html]: sorry, forgotten...

**Review by Tutt-J — APPROVED** [review]

**Pull request merged**